ajax/libs/scrollup/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "scrollup",
+  "filename": "jquery.scrollUp.min.js",
+  "version": "2.4.1",
+  "description": "A jQuery plugin to create a customisable 'Scroll to top' feature that will work with any website, with ease.",
+  "keywords": [
+    "jquery",
+    "plugin",
+    "scroll",
+    "scrollUp",
+    "scroll up",
+    "scroll to top",
+    "top",
+    "back to top"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/markgoodyear/scrollup"
+  },
+  "homepage": "https://markgoodyear.com/labs/scrollup/",
+  "author": {
+    "name": "Mark Goodyear",
+    "url": "https://markgoodyear.com/"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/markgoodyear/scrollup.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "jquery.scrollUp.*"
```

Comment:
Okey, I changed from `"jquery.scrollUp.*"` to `"**/jquery.scrollUp.*"`. Please review this PR again. Thanks.